packages/syntax-floating-components/src/Tooltip/Tooltip.stories.tsx

```diff
@@ -54,34 +55,65 @@ export default {
   tags: ["autodocs"],
 } as Meta<typeof Tooltip>;
 
+const proficiencyChoices = [
+  {
+    text: "Beginner",
+  },
+  {
+    text: "Intermediate",
+  },
+  {
+    text: "Advanced",
+  },
+];
+
 export const Default: StoryObj<typeof Tooltip> = {
   args: {
     delay: 0,
-    placement: "right",
-    initialOpen: true,
+    placement: "left",
+    initialOpen: false,
     strategy: "absolute",
-    children: "This is a tooltip",
+    children:
+      "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam",
   },
   render: ({ delay, placement, initialOpen, strategy, children }) => (
-    <div style={{ margin: "240px" }}>
-      <Tooltip
-        delay={delay}
-        placement={placement}
-        initialOpen={initialOpen}
-        strategy={strategy}
-      >
-        <TooltipTrigger>
-          <IconButton
-            accessibilityLabel="Info Icon Button"
-            icon={InfoOutlinedIcon}
-            onClick={() => alert("Default button pressed")}
-            color="tertiary"
-            size="lg"
-          />
-        </TooltipTrigger>
-        <TooltipContent>{children}</TooltipContent>
-      </Tooltip>
-    </div>
+    <Box paddingY={8} paddingX={4} backgroundColor="gray200">
```

Comment:
@JFChoe Could we add this as a separate example and keep the original as well? We want the default example to be the smallest and easiest example to copy/paste

Reply:
Alright, I made the radio group button example into it's separate example and reverted the default example to the original